addons/analytic/views/analytic_distribution_model_views.xml

```diff
@@ -21,14 +21,18 @@
             <form string="Analytic Distribution Model">
                 <sheet>
                     <group>
+                        <field name="analytic_distribution_stored_char" invisible="1"/>
+                        <span class="text-muted mt-4 mb-2">Simultaneous conditions to meet</span><br/>
```

Comment:
do we need the `mt-4` here?

Reply:
Not sure yet, I keep it because I had it there as benj was talking about adding a name.
He is going to rework the specs for this fix as we don't have nor will add a field name in 16.0 so I'm waiting for that